RegressionTests/runRegressionTests.sh

```diff
@@ -2,14 +2,17 @@
 
 echo "🦕🦕🦕🦕🦕🦕🦕🦕 Regression Testing has started 🦕🦕🦕🦕🦕🦕🦕🦕"
 
-muterdir="../../.build/debug"
-samplesdir="../../RegressionTests/samples"
+muterdir="../../../.build/debug"
+samplesdir="../../samples"
 
 rm -rf ./RegressionTests/samples
 mkdir -p ./RegressionTests/samples
 
+rm -rf ./RegressionTests/Repositories
+cp -R ./Repositories ./RegressionTests
+
 echo "Running Regression Test on BonMot..."
-cd ./Repositories/BonMot
+cd ./RegressionTests/Repositories/BonMot
```

Comment:
shouldn't we change that for the other too?

Reply:
I think only the first and last **cd** commands need to be changed. The remaining commands navigate within the RegressionTests directory.

Reply:
ah alright, they are using relative paths, cool!